modules/Q02_openai_question.sh

```diff
@@ -160,7 +160,8 @@ ask_chatgpt() {
                 local CNT=0
                 while [[ "${CNT}" -lt 1000 ]]; do
                   CNT=$((CNT+1))
-                  (( "${CNT}" % 100 == 0 )) && print_output "[*] Rate limit handling ... sleep mode - ${CNT}" "no_log"
+                  temp="$(( "${CNT}" % 100 ))"
```

Comment:
why this way?

Reply:
need to declare temp as local and capital pls.

```
local TMP_VAR=""
TMP_VAR=....
```

Reply:
> why this way?

This way semgrep does not skip the analysis